packages/panzer/adapters-ioss/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+TRIBITS_SUBPACKAGE(AdaptersIOSS)
+
+# Optional IOSS dependency
+##########################
+IF(${PACKAGE_NAME}_ENABLE_SEACASIoss)
+   GLOBAL_SET(PANZER_HAVE_IOSS ON)
+ELSE()
+   GLOBAL_SET(PANZER_HAVE_IOSS OFF)
+ENDIF()
+
```

Comment:
You are correct. We also don't need the if part of the if/else. I added a commit to remove it.